views/js/e2e/runner/_cleanup/cleanupCommands.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+import cleanupData from './cleanupData';
+import cleanupSelectors from './cleanupSelectors';
+
+/**
+ * Cleanup Commands
+ */
+Cypress.Commands.add('deleteImportedTest', () => {
+    
+    // Register route for test deletion
+    cy.route('POST', '**/delete').as('deleteTest');
+
+    // Visit Tests page
+    cy.visit(cleanupData.testsPageUrl);
+
+    // Select e2e example test
+    cy.get(cleanupSelectors.testsPage.treeManageTests).contains('e2e example test').click();
```

Comment:
The test failed for me at this point (the element is not loaded on the page yet). Consider adding a `cy.wait(event)` command before (it could be useful every time we do a `cy.visit()`).

Reply:
Fixed.

Reply:
It still fails at the same point for me, I think we still need a `cy.wait('@editClassLabel');` in `deleteImportedTest()` just like the one you added in `deleteDelivery()`.

Reply:
added